### PR TITLE
Smoke Tests (fixes #51)

### LINKED ISSUE #51: Make Smoke Test(s)

### PR DESCRIPTION
**Pull request by ravenmaster001**
Currently smoke tests for ensuring general usage of kordac is still functional and that docs are still building. If you have any other tests that you think are currently important please comment them for me todo.

### CODE REVIEW

**Comment by ravenmaster001** [comment]
@JackMorganNZ So I think pull-request checks test to see what happens if merged with develop.

File: kordac/tests/SmokeTests.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import unittest, os, subprocess
+from kordac import Kordac
+
+class SmokeDocsTest(unittest.TestCase):
+    """Tests opening of files and that kordac generates some output."""
```
Comment (JackMorganNZ) [inline kordac/tests/SmokeTests.py]: Wrong docstring

**Comment by JackMorganNZ** [comment]
Should test file names be singular or plural?

**Comment by ravenmaster001** [comment]
@JackMorganNZ Plural if containing multiple test classes or singular is one test class. If a file gets too big due to too many test classes it should be split into multiple files.

**Comment by JackMorganNZ** [comment]
Just realised this and came back to amend my comment, but have seen you pointed it out. Cheers.

**Pull request merged**